src/MSALWrapper/MSALWrapper.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <!-- Package Naming, Building, & Versioning -->
     <PackageId>Microsoft.Authentication.MSALWrapper</PackageId>
-    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>

```

Comment:
Since this is a library I'd prefer we not change this unless we have a good reason to. net6.0 is capable of consuming both netstandard2.0 and 2.1. We could change to just netstandard2.1.
Basically libraries should always target netstandard versions - which define the dotnet interface they require, and then any number of actual target frameworks for applications can consume it.